doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+# Controls and Shortcuts #
+
+The following is a list of input device controls and shortcuts for manipulating Gaffer's _Node Graph_, _Viewer_, and _Script Editor_ tabs.
+
+
+## _Node Graph_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Node Graph_ tab.
+
+### Node Creation ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Show node menu                        Right-click
+"                                     :kbd:`Tab`
```

Comment:
The " confused me. Can we list the action twice? Or separate Shortcut and Gesture into their own columns so we only need one row for this action?

Reply:
Best solution would be to convert these to HTML to enable multi-line cells, but I would want @johnhaddon to sign off on that.

Adding more tabs will widen the whole table, which we don't have the space for. Listing it twice would be bad table design. This form of reST table doesn't let you use empty cells. I could convert these to reST's CSV tables, but they're not much less readable than HTML tables, which would confer even more advantages.

Reply:
If the "" is confusing, isn't that bad table design too?